documentation/modules/ROOT/pages/02-architecture.adoc

```diff
@@ -1,104 +1,255 @@
-= Solution Pattern: Name Template
+= Building Apps around the Event Mesh
 :sectnums:
 :sectlinks:
 :doctype: book
 
 = Architecture 
 
-Introduction for the architecture of this solution pattern.
+The architecture of an event mesh-enabled system is a paradigm shift from traditional transactional designs to an eventual consistency model.
+This design aligns better with real-world processes, where different parts of a system may operate asynchronously yet collaboratively.
 
-== Common Challenges 
+In this section, we explore the technologies, flow, and structure that make the event mesh architecture resilient, scalable, and developer-friendly.
 
 [#tech_stack]
 == Technology Stack
 
-// Change links and text here as you see fit.
+Here's the list of technologies used in this solution and its examples:
+
 * Red Hat supported products
 ** https://www.redhat.com/en/technologies/cloud-computing/openshift[Red Hat OpenShift]
-** Red Hat Application Foundation
-*** https://access.redhat.com/products/quarkus[Quarkus]
-*** https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-streams-for-apache-kafka[Kafka Streams]
-* Other open source products:
-** https://www.postgresql.org/[PostgreSQL database]
-** https://helm.sh/[Helm]
-
+&mdash; Orchestrate containerized applications.
+Based on https://kubernetes.io/[Kuberentes].
+** https://www.redhat.com/en/technologies/cloud-computing/openshift/serverless[Red Hat OpenShift Serverless]
+&mdash; Provides the _Event Mesh_ and _Serverless_ capabilities.
+Based on https://knative.dev[Knative] project.
+** https://swc.saas.ibm.com/en-us/redhat-marketplace/products/red-hat-amq[Red Hat AMQ Streams]
```

Comment:
I think the name of the product is now "Streams for Apache Kafka": https://developers.redhat.com/products/streams-for-apache-kafka/overview

Reply:
Yes, I was thinking the same, but this is so confusing to me. 

I read somewhere it's, in fact, the name of the managed Kafka solution, but it looks it's aligning to replace the name. That's why I put IBM page there.

Anyway, I'll replace the reference with _Streams for Apache Kafka_ and link you provided.